src/chrome/content/rules/Geekzone.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="geekzone.co.nz (partial cdn)">
+	<target host="cdn.geekzone.co.nz" />
+	<target host="images.geekzone.co.nz" />

```

Comment:
GitHub strips HTML comments from a comment when you don't format them as code ;)
If you are looking for examples you should check out our [CONTRIBUTING.md](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md).